docs/pages/contributing.md

```diff
@@ -149,7 +149,7 @@ The following steps will get you setup to contribute changes to this repo:
    cd remix
 
    # if you are making *any* code changes, make sure to checkout the dev branch
-   git checkout dev
+   git checkout -b dev
```

Comment:
This creates a new branch called `dev` locally.
This is not what we want here, as we want the user to checkout the existing `dev` branch from the repo

Reply:
```suggestion
   git checkout dev
```